src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.*; 
+
+/**
+ * Represents tagging in the address book.
+ * Guarantees: immutable
+ */
+public class Tagging {
+	
+	private char _addOrRemoveSign;
+	private final Person _person;
```

Comment:
Good job on following the convention to add underscore to private class members. 👍